about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="logLine">
+            <ul>
+                <li class="icon"><img src="images/Twitter.png" alt="Twitter"></li>
+                <li class="icon"><img src="images/Github.png" alt="Github"></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log Out</a></li>
+            </ul>
+        </div>
+        <div class="menuIcon">
+            <span class="ban"></span>
+            <span class="ban"></span>
+            <span class="ban"></span>
+        </div>
+        <nav>
+            <a href="index.html" id="logo"><img src="images/logo.png" alt="logo"></a>
+            <ul class="nav">
+                <li><a class="ml" href="about.html">About</a></li>
+                <li><a class="ml" href="#">Program</a></li>
+                <li><a class="ml" href="#">Join</a></li>
+                <li><a class="ml" href="#">Sponsor</a></li>
+                <li><a class="ml" href="#">News</a></li>
+                <li><a class="ml" href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="about">
+        <div class="top">
+            <h4 id="hello1">"Hello! Sharing world"</h4>
+            <h2 id="year">Different aspects of Afghanistan summit 2022</h2>
+            <span class="span"></span>
+            <p id="det">This summit is going to exchange the different aspects of Afghanistan in the past years and shows the world the real face. This summit is going to exchange the different aspects of Afghanistan in the past years and shows the world the real face. This summit is going to exchange the different aspects of Afghanistan in the past years and shows the world the real face.This summit is going to exchange the different aspects of Afghanistan in the past years and shows the world the real face</p>
+            <p id="please">Please contact us via email below</p>
+            <a href="#">Aliahmad.karimi1997@gmail.com</a>
+        </div>
+        <div class="center">
+            <h4>Different aspects of Afghanistan summit 2022 Logo </h4>
```

Comment:
- [ ]  Kindly add a divider after the "Different aspects of Afghanistan summit 2022 Logo" header, this will make your design look like the original one

|Your design|CC design|
|-|-|
|![Screenshot_20220804_065809](https://user-images.githubusercontent.com/25227022/182773847-f01eceaa-dcf6-4ae8-b5a1-6a9c7bafb4b0.png)|![Screenshot_20220804_065842](https://user-images.githubusercontent.com/25227022/182773862-b3f486b2-c9f4-4968-83b2-3ec4f5709c73.png)|